go.mod

```diff
@@ -1,28 +1,26 @@
 module github.com/IBM-Cloud/ppc-aas-go-client
 
-go 1.17
+go 1.18
```

Comment:
should update to 1.19 since 1.18 is at end of life

Reply:
Updated to golang 1.19